netty/src/test/java/io/grpc/netty/NettyStreamTestBase.java

```diff
@@ -166,7 +162,8 @@ public void notifiedOnReadyAfterWriteCompletes() throws IOException {
   public void shouldBeReadyForDataAfterWritingSmallMessage() throws IOException {
     sendHeadersIfServer();
     // Make sure the writes don't complete so we "back up"
-    reset(future);
+    ChannelPromise uncompletedPromise = new DefaultChannelPromise(channel);
+    when(writeQueue.enqueue(any(QueuedCommand.class), anyBoolean())).thenReturn(uncompletedPromise);
```

Comment:
This is not necessary if the suggestion in one of the other comments applied, but okay to keep it.